### PR TITLE
NEW adding button Send Email on the salary file

### PR DESCRIPTION
**Pull request by cmfpmatik**
Adding button to send salary to employee.

### CODE REVIEW

**Comment by eldy** [comment]
Can you provide a screenshot of what's new ?

**Comment by cmfpmatik** [comment]
> Can you provide a screenshot of what's new ?
sendmail_salary

File: htdocs/salaries/card.php [hunk 1]
```diff
@@ -662,9 +669,9 @@
 	print '<script>';
 	print '$( document ).ready(function() {';
 		print '$("#updateAmountWithLastSalary").on("click", function updateAmountWithLastSalary() {
+					console.log("We click on link to autofill salary amount url="+url);
```
Comment (eldy) [inline htdocs/salaries/card.php:672]: Why moving this line up ?
The line is nod before the line var url = ...
but this line include the value url into its log.

Comment (cmfpmatik) [reply htdocs/salaries/card.php:672]: Don't know why.
This is a probably mistake.

**Pull request merged**